web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -30,7 +30,8 @@
 public class WebUtil implements GlobalPropertyListener {
 	
 	private static final Logger log = LoggerFactory.getLogger(WebUtil.class);
-	
+
+	@Deprecated
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation